index.html

```diff
@@ -634,28 +632,28 @@ <h3>
               entering the URL in the user agent's address bar, or form
               submission, or initializing through a script operation other than
               the <a data-cite=
-              "!HTML/history.html#dom-location-reload">location.reload()</a>
+              "HTML/history.html#dom-location-reload">location.reload()</a>
               method, let the navigation type be the <code>DOMString</code>
```

Comment:
you can probably `<dfn data-cite="....">DOMString</dfn>` somewhere and point to the WebIDL spec.

Reply:
The all the DOMString can get auto linked.